app/src/test/kotlin/fr/free/nrw/commons/category/CategoriesModelTest.kt

```diff
@@ -17,12 +20,26 @@ class CategoriesModelTest {
     @Mock
     internal var categoryItem: CategoryItem? = null
 
+    @Spy
+    internal lateinit var gson: Gson
+
+    @Spy
+    internal lateinit var categoryItemForMethodA: CategoryItem
```

Comment:
@kbhardwaj123 'Method A' might be mentioned in the wiki but its not a term that all the contributors might be aware of. Infact even I have heard this term for the first time. Please rename it to something more meaningful. 

Rename other usages as well.

Reply:
Sure sir will do that